pkg/apis/policy/v1alpha1/upstreamtrafficsetting.go

```diff
@@ -38,6 +38,20 @@ type UpstreamTrafficSettingSpec struct {
 	// directed to the upstream host.
 	// +optional
 	ConnectionSettings *ConnectionSettingsSpec `json:"connectionSettings,omitempty"`
+
+	// RateLimit specifies the rate limit settings for the traffic
+	// directed to the upstream host.
+	// If HTTP rate limiting is specified, the rate limiting is applied
+	// at the VirtualHost level applicable to all routes within the
+	// VirtualHost.
+	// +optional
+	RateLimit *RateLimitSpec `json:"rateLimit,omitempty"`
+
+	// HTTPRoutes defines the list of HTTP routes settings
+	// for the upstream host. Settings are applied at a per
+	// route level.
+	// +optional
```

Comment:
To clarify, this doesn't actually do anything does it? It's purely informational?

Reply:
From a code perspective, it doesn't. But it renders the field as `optional` in the docs. E.g. https://release-v1-1.docs.openservicemesh.io/docs/api_reference/policy/v1alpha1/

![image](https://user-images.githubusercontent.com/21697719/172679054-5953be57-9c97-4b60-b105-a763456413ef.png)

Reply:
Oh that is VERY good to know; I had no idea. Will be doing this more often 👍🏾